nidirect_health_conditions/src/Controller/HealthConditionsListingController.php

```diff
@@ -54,6 +54,24 @@ public static function create(ContainerInterface $container) {
     );
   }
 
+  /**
+   * Controller callback for the page title.
+   *
+   * Use this to examine route parameters/any other conditions
+   * and vary the string that is returned.
+   *
+   * @return string
+   *   The page title.
+   */
+  public function getTitle($route_type) {
```

Comment:
If you've got a parameter here, it might be useful to use a type hint eg: `getTitle(string $route_type)`... this helps tools like PHPStorm build up a symbol/function library and assist in spotting small errors and aids in autocomplete.

It's not essential by any means... this will work, but it wouldn't necessarily be clear to me here whether $route_type should be a route name or some other value; object or something else for instance.

Reply:
Got a bit stuck with this Johan.  $route_type is basically a custom argument (https://www.drupal.org/docs/drupal-apis/routing-system/structure-of-routes#s-passing-arguments-to-controllers) specified in drupal8/web/modules/custom/nidirect_health_conditions/nidirect_health_conditions.routing.yml.  I've just set the type hint to be string?

Reply:
I'm sure that's fine! Not that it matters massively to PHP anyway.... something like '12' and 12 is suitably type-juggled and normally fine (compiled lang devs must have massive eye-twitches by now!)... there's weird stuff in core for this such as when an argument of 'node' sometimes is an object and other times is a node id so I don't think there's any huge concern here.